CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+v0.2.0:
```

Comment:
Is there a reason for this style? We use [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) everywhere else.

Reply:
Yes, I don't like that verbose style. This gets to the point with much less overhead.

I also include the changelog in each annotated tag, which is pretty much just `r ! head CHANGELOG` with this format.

Reply:
Hmm okay I don't think this is the appropriate way to change how our changelogs looks like and I think you know that too :p 

I prefer consistency over personal preferences. If we all agree this format is preferred, then we should change the other changelogs (I guess a script for that isn't that difficult?). Has this been discussed?

Reply:
I don't think consistency very important for changelogs for the same reason it is not important for READMEs: they are files for human consumption, not for machines.

Reply:
I made it a little closer now: it uses markdown headers and I added the date.

Reply:
I also don't think this changelog should be included if we give update notes to customers, so I don't think it matters for that either. This package doesn't change product features that are interesting for the customers at all.

Reply:
I'd still insist on having consistency, versus everybody having their own format. If this is the format to use, then that's fine by me, but we should change it everywhere. @vsuryamurthy any opinion?

Reply:
I agree with Hans in this case, it should be consistent over repositories. Since the rust repositories to follow would take inspiration from this particular repository, I foresee two different formats popping up thereby resulting in confusion which should be avoided.

Reply:
I still disagree, I think this format adds a lot of pointless noise that makes it harder to read the changelog. But w/e, I'm not updating hundreds of changelogs.